docs/sdks/sdk-ref-javascript.md

```diff
@@ -319,14 +320,33 @@ The `opts` object supports the following properties.
 | `useCookie` | `boolean` | Optional | Set this to `true` to tell the SDK to store the identity in cookie storage instead of local storage. You can still provide an identity using a first-party cookie if this value is false or not provided. | 
 | `callback` | `function(object): void` | Deprecated | The function that the SDK should invoke after validating the passed identity. Do not use this for new integrations. | N/A |
 
+#### Multiple Init Calls
+
+The `init()` function can be called any number of times.  The code will, in most cases, accept the latest value of a certain init parameter. For example, if init is called twice, and a different `baseUrl` is passed in each call, the `baseUrl` variable will be updated to its value in the second call. 
+
+There are two exceptions to this functionality:
+
+1. If a new identity is passed in a subsequent call, and the new identity expires before the current identity, the new identity will not replace the current identity.  
+2. For every subsequent callback function passed, it is added to the existing array of callbacks using the [Array Push Pattern](#array-push-pattern).
+
+Note: If `useCookie` is updated, the location of the identity will change - i.e. if it is updated from `true` to `false`, the first party cookie will be removed and the identity added to local storage.
```

Comment:
Use Docusaurus note coding plus present tense. Suggest:

:::note
If `useCookie` is updated, the location of the identity changes. For example, if this value is updated from `true` to `false`, the first-party cookie is removed and the identity is added to local storage.
:::